dashboard/src/features/app/components/Navigation/Navigation.jsx

```diff
@@ -56,7 +56,7 @@ class Navigation extends React.Component {
 
         <ul className={styles.navigation}>
           <li className={styles.navigationTitle}>services</li>
-          {!this.props.production &&
+          {!this.props.production && this.props.mockhsm &&
```

Comment:
I was trying to touch as little as necessary, but I think it makes sense to remove it now.